android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -49,6 +52,7 @@
     private GoogleMap mMap;
     private Marker locationMarker;
     private List<String> searchItems = new ArrayList<>();
+    private SharedPreferences sharedPreferences;
```

Comment:
@iamareebjamal 
i am using the Utils class then and creating getters and setters to get the value? is it fine?